plugins/decklink/DecklinkBase.cpp

```diff
@@ -15,4 +15,8 @@ bool DecklinkBase::Activate(DeckLinkDevice *, long long)
 	return false;
 }
 
-void DecklinkBase::Deactivate() {}
```

Comment:
For consistency I'd keep this in the file instead of the header like all the other functions.

Reply:
The reason I made this choice is the other methods have default returns of` false`. 
This is a no-op and must be implemented by the class above. Making it clearly abstract/nop in the header can prevent this oversight to clarify that it should be overridden.

Reply:
If you want to make it abstract (pure virtual, force others to implement) you should do `virtual void Deactivate() = 0;`
`virtual void Deactivate(){};` creates a default implementation with an empty body.
(Either way it's not a big deal, just pointing it out.)

Reply:
GAH true that's what I meant to do. Might mend in a few